cmd/titus-storage/fs.go

```diff
@@ -0,0 +1,121 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os/exec"
+	"strings"
+
+	"github.com/Netflix/titus-executor/logger"
+)
+
+func isMkfsNeeded(ctx context.Context, device string, fstype string) (bool, error) {
+	l := logger.GetLogger(ctx)
+	existingFormat, err := getDiskFormat(ctx, device)
+	if err != nil {
+		return false, err
+	}
+	if existingFormat == "" {
+		return true, nil
+	}
+	l.Warnf("Existing format on disks detected, no mkfs needed: '%s'", existingFormat)
+	return false, nil
+}
+
+func mkfs(ctx context.Context, device string, fstype string) error {
+	if fstype == "ext4" || fstype == "ext3" || fstype == "" {
+		args := []string{
+			"-F",  // Force flag
+			"-m0", // Zero blocks reserved for super-user
+			device,
+		}
+		return runMkfs(ctx, fstype, device, args)
```

Comment:
What happens when fstype == "", wont it run "mkfs."

Reply:
err..  good point...